app/domain/regulations_per_day.py

```diff
@@ -167,8 +167,12 @@ def check_min_daily_rest(
                     0
                 ].start_time + timedelta(days=1)
                 extra["breach_period_end"] = breach_period_end.isoformat()
-                extra["breach_period_max_break_in_seconds"] = (
-                    breach_period_end - long_break.start_time
+                extra[
+                    "breach_period_max_break_in_seconds"
+                ] = get_longest_inner_break(
+                    activities_related_to_long_break,
+                    long_break,
+                    breach_period_end,
```

Comment:
:orange_circle: On est sûr que cette méthode va toujours retourner une valeur not None ?